salt/metalk8s/repo/configured.sls

```diff
@@ -1,2 +1,2 @@
 include:
-  - .offline
+  - .{{ grains['os'] }}
```

Comment:
Here I think you should also (like below) filter on `osfamily` I think if we support RedHat we will have almost the same states for both.

Reply:
Change done.